gp-nested-forms/gpnf-hide-quantity-in-summary-field-values.php

```diff
@@ -0,0 +1,17 @@
+<?php
+/**
+ * Gravity Perks // Nested Forms // Hide Quantity in Summary Field Values
+ * https://gravitywiz.com/documentation/gravity-forms-nested-forms/
+ *
+ * Product field display values are formatted as "Label, Quantity, Price". This snippet removes the quantity from the display value.
```

Comment:
Are there any cases where the order could change? E.g. is that order configurable?

A more specific question is: is there a config somewhere we could grab and dynamically build the regex from that?

I wouldn't sink too much time into this, if any. More so just a question in case it's an easy thing to add